bindings/wasm/manifold.d.ts

```diff
@@ -465,9 +465,7 @@ declare interface ManifoldStatic {
   getCircularSegments: typeof getCircularSegments;
   ManifoldFromMeshVec: typeof ManifoldFromMeshVec;
   Manifold: typeof Manifold;
-  Vector_vec3: typeof Vector_vec3;
-  Vector_vec4: typeof Vector_vec4;
```

Comment:
@rafern I don't think these are necessary, but maybe I'm missing something?

Reply:
I added them because they're not just plain JS objects that need to follow that interface; they need to be created via a constructor. Since all the classes and methods that are only available after importing were put in ManifoldStatic, I also added those.

Reply:
Okay, well TS doesn't like them, and I will likely remove this anyway as you said, so good enough for now.